src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -5640,6 +5640,9 @@ public function seek( $bookmark_name ): bool {
 	 * @return bool Whether the bookmark was successfully created.
 	 */
 	public function set_bookmark( $bookmark_name ): bool {
+		if ( $this->is_virtual() ) {
```

Comment:
Should this include a `_doing_it_wrong`?

Reply:
It seems like it would be helpful. There is a similar condition that marks an error with `_doing_it_wrong`, so I would replicate it here, too:

https://github.com/WordPress/wordpress-develop/blob/0d428194d1e6d6f1285d45cafd84b18f514070c1/src/wp-includes/html-api/class-wp-html-tag-processor.php#L1340-L1345

Reply:
I added the message 👍